src/app/components/ms-contents/ms-contents.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="msContents">
+    <evt-ms-desc-section [label]="'msContents'"></evt-ms-desc-section>
+    <div>
+        <ng-container *ngIf="data?.structuredData; else elseBlock">
+            <div class="section">
+                <div class="row no-gutters" *ngIf="data?.summary">
+                    <div class="col-1">
+                        <evt-ms-desc-section [label]="'summary'" [inlineLabel]="true"></evt-ms-desc-section>
+                    </div>
+                    <div class="col-6 col-content">
+                        <evt-content-viewer [content]="data.summary" content></evt-content-viewer>
+                    </div>
+                </div>
+                <div class="row no-gutters" *ngIf="data?.textLang.length>0">
+                    <div class="col-1 col-label">
+                        <evt-ms-desc-section [label]="'textLang'" [inlineLabel]="true"></evt-ms-desc-section>
+                    </div>
+                    <div class="col-lg-10 col-content">
+                        <div content>
+                            <span *ngFor="let tl of data.textLang | keyvalue">
+                                <evt-content-viewer [content]="data.textLang[tl.key]"></evt-content-viewer>
+                                <span *ngIf="tl.key < [data.textLang.length-1]">, </span>
+                            </span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <evt-ms-desc-section [label]="'items'" [inlineLabel]="true"></evt-ms-desc-section>
+            <ng-container *ngFor="let m of data?.msItem">
+                <evt-ms-item [data]="m" [nested]="false"></evt-ms-item>
+                <span *ngIf="m?.msItem.length>0">
+                    <evt-ms-item *ngFor="let ms of m.msItem" [data]="ms" [nested]="true"></evt-ms-item>
+                    <ng-container *ngFor="let ms of m?.msItem">
+                        <evt-ms-item *ngFor="let item of ms.msItem" [data]="item" [nested2]="true"></evt-ms-item>
+                    </ng-container>
+                </span>
+            </ng-container>
+        </ng-container>
+        <ng-template #elseBlock>
+            <div class="container">
+                <div class="row">
+                    <div class="col-1 col-label">
+                        <evt-ms-desc-section [label]="'summary'" [inlineLabel]="true"> </evt-ms-desc-section>
+                    </div>
+                    <div class="col-6 col-content">
```

Comment:
Why do you use "col-6" and "col-content"? Anyway, this information will be contained in a panel, so you should use all the available space.